src/test/groovy/org/prebid/server/functional/util/PBSUtils.groovy

```diff
@@ -43,8 +44,8 @@ class PBSUtils implements ObjectMapperWrapper {
         createTempFile(data, ".json")
     }
 
-    static BigDecimal getRandomFloorValue() {
-        roundDecimal(getRandomDecimal(FLOOR_MIN, 2), 2)
+    static BigDecimal getRandomFloorValue(float floorMin = FLOOR_MIN, float floorMax = FLOOR_MAX) {
```

Comment:
If this method is only used in descendants of the `src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsBaseSpec.groovy`, then move it there. It's strange when you reference external constants from base test class in utility class.

Reply:
Ugh, ok. I also see that we are using that method in a model too. Will need to think how to refactor that.